recipes/default.rb

```diff
@@ -18,11 +18,15 @@
 
 pg = Chef::EncryptedDataBagItem.load(node['timesync']['databag'],
                                      'pg')
+secret_key = Chef::EncryptedDataBagItem.load(node['timesync']['databag'],
+				     'key')
 
 environment = {
   'PG_CONNECTION_STRING' => "postgres://#{pg['user']}:#{pg['pass']}@" \
     "#{pg['host']}:#{pg['port']}/#{pg['database_name']}",
-  'NODE_ENV' => 'production'
+  'NODE_ENV' => 'production',
+  'SECRET_KEY' => secret_key['key'],
+  'INSTANCE_NAME' => 'osuosl-timesync-staging'
```

Comment:
I don't think we should hard code this in if we want to use this cookbook for deploying to production some day. Could you set this as an attribute? it would look something like:

in attributes/default.rb:
```
default['timesync']['instance_name'] = 'osuosl-timesync-staging'
```
and here:
```
'INSTANCE_NAME' => node['timesync']['instance_name']
```